### ISSUE TITLE
[antd, react-doc-viewer] Failed to load module script: The server responded with a non-JavaScript MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec.

### ISSUE THREAD

**Issue by Relaxe111** [pikapkg/cdn#111]
When i use streaming imports in snowpack and try to add react-doc-viewer and antd i get following errors: 
`Failed to load module script: The server responded with a non-JavaScript MIME type of "application/octet-stream". Strict MIME type checking is enforced for module scripts per HTML spec`
url for the antd is:
`http://localhost:8081/_snowpack/pkg/error/build:antd@v4.10.2-brvSpWOme5M2tl48XtzZ`
minimal project to reproduce is here: [ https://github.com/Relaxe111/snowpack3-csa-react-ts-antd-source-remote.git]( https://github.com/Relaxe111/snowpack3-csa-react-ts-antd-source-remote.git)
url for the react-doc-viewer is:
`http://localhost:8081/_snowpack/pkg/-/pdfjs-dist@v2.4.456-WBzJ8pEGmpogsCfpmUqJ/dist=es2020,mode=raw/build/pdf.worker.entry`
minimal project to reproduce issue is: 
[https://github.com/Relaxe111/snowpack-react-ts-doc-viewer.git](https://github.com/Relaxe111/snowpack-react-ts-doc-viewer.git)

Is there something wrong with streaming imports or problem lies within packages itself?

**Comment by FredKSchott** [comment]
Just fixed this in 76dca31aa0518d5e672e5f81ef272726db420202 , thanks for reporting!

**Comment by Relaxe111** [comment]
I still get same error. but now when i opened the file if found there following:
```
http://localhost:3000/_snowpack/pkg/error/build:antd@v4.10.3-HOWJ2WcmOqfRkRYZifDh
/*
 * [Package Error] "antd@v4.10.3" could not be built. 
 *
 *   Installing package...
 *   Could not resolve "rc-util/es/omit".
 *
 * How to fix:
 *   - If you believe this to be an error in Skypack, file an issue here: https://github.com/pikapkg/cdn/issues
 *   - If you believe this to be an issue in the package, share this URL with the package authors to help them debug & fix.
 *   - Use https://pika.dev/ to find a web-friendly alternative to find another package.
 */

console.warn("[Package Error] \"antd@v4.10.3\" could not be built. \nInstalling package...\nCould not resolve \"rc-util/es/omit\".");
throw new Error("[Package Error] \"antd@v4.10.3\" could not be built. ");
export default null;
```
